src/server/context.ts

```diff
@@ -253,8 +253,8 @@ export class ServerContext {
           component,
           handler,
           csp: Boolean(config?.csp ?? false),
-          appLayout: Boolean(config?.appTemplate ?? true),
-          rootLayout: Boolean(config?.rootLayout ?? false),
+          appTemplate: Boolean(config?.appTemplate ?? true),
+          inheritLayouts: Boolean(config?.inheritLayouts ?? true),
```

Comment:
True by default is very confusing. Can we invert the naming so it is `false` by default? Same for `appLayout`.

Reply:
https://www.w3.org/TR/design-principles/#optional-parameters